lib/js/index.js

```diff
@@ -342,7 +349,7 @@ class DateTimePicker extends Events {
   setDate(date) {
     const m = moment(date);
     const month = m.format('MMM');
-    const day = m.format('Do');
+    const day = m.format('D');
```

Comment:
Whilst this change better matches the material "picker" examples, I think it makes the component feel visually unbalanced.

Can you add options for the number strings instead?